NAMESPACE

```diff
@@ -13,7 +13,6 @@ export(release)
 export(tag_version)
 export(unbump_version)
 export(update_news)
-export(update_version)
```

Comment:
if we agree on the strategy, this should be deprecated, not just removed :innocent: